tools/kubectl-ownerref/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+var (
+	ns = flag.String("ns", "", "namespace")
+)
+
+func main() {
```

Comment:
We probably want to evolve this to use the shared code from cli-runtime, for example https://github.com/kubernetes/sample-cli-plugin